### PR TITLE
dev cinia

### PR DESCRIPTION
**Pull request by ciniiia**
### Changelog / Description
- Add lab detail address in customer request test by find lab
- Change view to delete in lab service detail page
- Add dialogs (in 2 conditions) when customer failed to checkout
- Add rating lab in fulfilled order
- fix route in order displayed in dashboard from order history to order detail
- take out forgot password when sign in
- add page request test
- add image validation when add and edit service

### CODE REVIEW

File: src/components/DialogSelectUserLogin.vue [hunk 1]
```diff
@@ -26,13 +26,13 @@
                 @keyup.enter="onPasswordInput"
               ></v-text-field>
 
-              <div
+              <!-- <div
```
Comment (kevinjanada) [inline src/components/DialogSelectUserLogin.vue:29]: comment ini dhapus aja mba

File: src/components/DialogSelectUserLogin.vue [hunk 2]
```diff
@@ -228,15 +228,15 @@ export default {
...
-    forgotPassword() {
-      const dataPair = JSON.parse(this.keystore);
-      this._show = false;
-      this.$emit("forgot-password", {
-        status: true,
-        address: dataPair[0].address,
-      });
-      this.clearInput();
-    },
+    // forgotPassword() {
```
Comment (kevinjanada) [inline src/components/DialogSelectUserLogin.vue:231]: dihapus aja ya

File: src/router/routes/customer.js [hunk 3]
```diff
@@ -114,6 +114,25 @@ const customerRoutes = [{
...
       component: () => import(/* webpackChunkName */ '../../views/Dashboard/Customer/RequestTest')
     },
+    {
+      path: '/customer/request-test/find-lab',
+      name: 'find-lab',
+      meta: {
+        pageHeader: 'Find a Lab',
+        breadcrumbs: [
+          { text: 'Request a Test', href: '/customer/request-test' },
+          // { text: 'Checkout', disabled: true },
```
Comment (kevinjanada) [inline src/router/routes/customer.js:124]: ini juga dihapus aja ya

**Review by kevinjanada — COMMENTED** [review]
Minta tolong yang di comment ya mba

File: src/components/DialogSelectUserLogin.vue (hunk 1 above)
Comment (ciniiia) [reply src/components/DialogSelectUserLogin.vue]: done

File: src/components/DialogSelectUserLogin.vue (hunk 2 above)
Comment (ciniiia) [reply src/components/DialogSelectUserLogin.vue]: done

File: src/router/routes/customer.js (hunk 3 above)
Comment (ciniiia) [reply src/router/routes/customer.js]: done

**Review by kevinjanada — APPROVED** [review]

**Pull request merged**